src/model/image-tag.test.ts

```
@@ -8,9 +8,13 @@ describe('ImageTag', () => {
       expect(() => new ImageTag(unityVersion)).not.toThrow();
     });
 
-    test.each(['2000.0.0f0', '2011.1.11f1'])('accepts %p version format', (version) => {
+    test.each(['2000.0.0f0', '2011.1.11f1', '6000.0.0f0'])('accepts %p version format', (version) => {
```

ESLint

[error] Replace `'accepts·%p·version·format',` with `⏎······'accepts·%p·version·format',⏎·····`

(prettier/prettier)